### DISCUSSION TITLE
Automatic Discord Rich Presence (RPC) Integration Based on Running Apps

### DISCUSSION THREAD

**Discussion by shiftyrun** [maximmax42/Discord-CustomRP#251] [Ideas]
### **Is your feature request related to a problem? Please describe.**
I often find it cumbersome to manually select presets and start Discord RPC when specific apps are running.

### **Describe the solution you'd like**
I want a feature that automatically loads a preset and starts Discord RPC when a specific app is detected running. This removes the need to manually select a file. If multiple instances of the same app are detected, the feature will prioritize the first instance alphabetically based on the process name. It will also notify the user via a Windows notification if multiple instances are detected.

### **Describe alternatives you've considered**
I haven't considered any alternatives yet.

### **Additional context**
No additional context at this time.

**Comment by shiftyrun** [comment]
...Am I asking too much?

**Comment by maximmax42** [comment]
I mean, yeah, it's not a bad idea. Shouldn't be that hard. Maybe in the future.

**Reply by shiftyrun** [reply]
Thank you! Have a wonderful day.

**Comment by MH285** [comment]
Had the exact same thought. Would be AMAZING

**Comment by Gills2794** [comment]
Would love this feature addition! I can see myself assigning it to a few games where I can choose custom icons instead of the default game icon

**Comment by zytancl** [comment]
just starting using this and i want this so bad too! pls add this feature!
